corehq/apps/app_manager/views/apps.py

```diff
@@ -277,12 +277,6 @@ def app_source(request, domain, app_id):
 
 
 @require_can_edit_apps
-def copy_app_check_domain(request, domain, name, app_id_or_source):
```

Comment:
I don't. Looked at history, it's been its own view for at least a year, but not sure why.